src/services/search-service.js

```diff
@@ -1,6 +1,30 @@
 import qs from 'qs';
 import axios from 'axios';
 
```

Comment:
The new default parameters are lifted from Kent's email, with values that are already defaults removed.

Reply:
I'll add some comments to this indicating what parameters do.